### PR TITLE
Log_softmax is a member of torch and torch.nn.functional.

### PR DESCRIPTION
**Pull request by adityaiitb**
Signed-off-by: Aditya Agrawal <aditya.iitb@gmail.com>

### CODE REVIEW

File: pyprof/prof/softmax.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def __init__(self, d):
         self.op_ = op
         self.args = args
 
-        assert (mod == "torch.nn.functional")
+        assert (mod in ["torch", "torch.nn.functional"])
```
Comment (debermudez) [inline pyprof/prof/softmax.py:89]: Can you link the documentation for torch.log_softmax?
I am not finding it.

Comment (adityaiitb) [reply pyprof/prof/softmax.py:89]: It exists but seems to be undocumented, which is probably why I never encountered it before.

Comment (debermudez) [reply pyprof/prof/softmax.py:89]: verified by pytorch team
